src/main/java/gift/controller/AdminController.java

```diff
@@ -29,15 +32,16 @@ public AdminController(ProductService productService) {
   }
 
   @GetMapping
-  public String productList(Model model, @PageableDefault(size = 5, sort = "id") Pageable pageable) {
-    Page<ProductResponseDto> productPage = productService.findAllProduct(pageable);
+  public String productList(Model model, @PageableDefault(size = 5, sort = "id", direction = Sort.Direction.ASC) Pageable pageable) {
+    Pageable fixedPageable = PageRequest.of(pageable.getPageNumber(), 5, pageable.getSort());
+    Page<ProductResponseDto> productPage = productService.findAllProduct(fixedPageable);
     model.addAttribute("productPage", productPage);
     return "admin/list"; // templates/admin/list.html
   }
 
   @GetMapping("/add")
   public String addForm(Model model) {
-    model.addAttribute("product", new ProductRequestDto("", BigDecimal.ZERO, "", false));
+    model.addAttribute("product", new ProductRequestDto("", BigDecimal.ZERO, "", false, Collections.emptyList()));
```

Comment:
이런 케이스는 비어있음을 나타내는 EMPTY 상수를 만들고 사용해봐도 괜찮을 것 같아요. 😃 

 ```suggestion
    model.addAttribute("product", ProductRequestDto.EMPTY);
```

Reply:
이런 방식으로 구현할 생각을 아예 못했었는데, 새로운 시각으로 볼 수 있게 해주셔서 감사합니다 !
ProductRequestDto에 EMPTY 상수를 따로 구현했습니다 !